include/rsp_queue.inc

```diff
@@ -158,6 +158,9 @@ _RSPQ_SAVED_STATE_END:
 # except to read command arguments.
 #define rspq_dmem_buf_ptr gp
 
+# Is set to the command's size in bytes when jumping to the command function
+#define rspq_cmd_size t7
```

Comment:
It becomes public API: it's where each command can read its size, if needed. It can be used to reuse implementation against different-sized commands, or in the future to allow for variable-sized commands.

Reply:
Is it safe to use that register in custom ucode if it doesn't need that information? Still we should document it IMHO.

Reply:
Agreed, I've improved the docs